app-crypt/asekey/files/asekey-3.7-musl-ushort.patch

```diff
@@ -0,0 +1,10 @@
+--- a/Ase.h	2012-12-16 21:46:48.000000000 +0200
```

Comment:
Please add a comment at the top explaining why you're doing the hax instead of a full fix.

Also, scrub-patch from iwdevtools.

Reply:
Hopefully rationale is sufficient.